joomla/joomla-cms
# [4.3] Migrate quickicon extensionupdate plugin to services

### Summary of Changes

Migrate the quickicon extensionupdate plugin to service providers and use SubscriberInterface.

### Testing Instructions

- Install an older version of an extension with providing an update server
- Check if quickicons are still working

![image](https://user-images.githubusercontent.com/66922325/200319856-d528569d-3e5f-4e34-b4f6-38e1bccb528c.png)
![image](https://user-images.githubusercontent.com/66922325/200319878-1cf2c453-9759-4f2f-8207-d121608a565f.png)

### Actual result BEFORE applying this Pull Request

Extensionupdate quickicon is working

### Expected result AFTER applying this Pull Request

Extensionupdate quickicon is working (and the other quickicons are still displayed)

### Link to documentations
Please select:
